tests/passes/cell-share/bounded.expect

```diff
@@ -1,9 +1,10 @@
+{"main":{"std_add_WIDTH_32":{"2":6},"std_mult_pipe_WIDTH_32":{"6":6},"std_reg_WIDTH_32":{"2":2,"4":4}}}
```

Comment:
This is no longer a valid Calyx file. Should we try to make sure we still generate a valid Calyx file when printing out both the file and the JSON information out? We could use `log::info!` or something but that makes it hard to print pipe the JSON output into something else.

Maybe the `print-share-freqs` can take `stderr` and `stdout` as an option and print to the respective stream

Reply:
Not sure if I'm understanding 100 percent correctly, but rn, instead of taking in `cmd`, you can pass in `file-name` and it prints the json information out into the file named `file-name`. So we can always have access to both the json information and a valid calyx file. The reason I printed it out with the calyx file for the test was just for convenience. 

Not sure if I'm completely answering your question though.